features/apimgt/org.wso2.carbon.apimgt.publisher.feature/src/main/resources/publisher/source/src/app/components/Apis/Details/Resources/components/operationComponents/AddParameter.jsx

```diff
@@ -80,6 +84,18 @@ function AddParameter(props) {
     React.useEffect(() => {
         setLabelWidth(inputLabel.current.offsetWidth);
     }, []);
+
+    React.useEffect(() => {
```

Comment:
@kavishkafernando , I think we don't need an `effect` here ? we could derive the `isParameterExist` from the `operation` props

Reply:
fixed